tools/benchmark/cmd/put.go

```diff
@@ -68,40 +68,38 @@ func putFunc(cmd *cobra.Command, args []string) {
 
 	for i := range clients {
 		wg.Add(1)
-		go doPut(clients[i], requests)
+		go doPut(context.Background(), clients[i], requests)
 	}
 
-	start := time.Now()
-	for i := 0; i < putTotal; i++ {
-		r := &etcdserverpb.PutRequest{
-			Key:   k,
-			Value: v,
+	pch := printReport(results)
```

Comment:
pdoneC seems to be more descriptive. So reader can know why we wait for the signal from it without reading the function implmentation.